core/src/main/java/org/dozer/loader/xml/DozerResolver.java

```diff
@@ -42,12 +43,18 @@
 public class DozerResolver implements EntityResolver {
 
     private final Logger log = LoggerFactory.getLogger(DozerResolver.class);
+    private static final String VERSION_5_XSD = "http://dozer.sourceforge.net/schema/beanmapping.xsd";
+    private static final String VERSION_6_XSD = "http://dozermapper.github.io/schema/bean-mapping.xsd";
 
     public InputSource resolveEntity(String publicId, String systemId) {
         InputSource source = null;
 
         log.debug("Trying to resolve XML entity with public ID [{}] and system ID [{}]", publicId, systemId);
 
+        if (VERSION_5_XSD.equalsIgnoreCase(systemId)) {
+            throw new MappingException("Found v5 XSD: '" + VERSION_5_XSD + "'. Expected v6 XSD: '" + VERSION_6_XSD + "'");
```

Comment:
I would suggest to add a more explicit hint for end-user in exception message. For instance, telling them that starting from v6 new schema location is used and all users must upgrade their mapping files.